dotnet/src/SemanticKernel.Abstractions/Contents/FunctionCallContentBuilder.cs

```diff
@@ -40,7 +41,29 @@ public void Append(StreamingChatMessageContent content)
     /// Builds a list of <see cref="FunctionCallContent"/> out of function call updates tracked by the <see cref="Append"/> method.
     /// </summary>
     /// <returns>A list of <see cref="FunctionCallContent"/> objects.</returns>
+    [RequiresUnreferencedCode("Uses reflection to deserialize function arguments, making it incompatible with AOT scenarios.")]
+    [RequiresDynamicCode("Uses reflection to deserialize function arguments, making it incompatible with AOT scenarios.")]
     public IReadOnlyList<FunctionCallContent> Build()
```

Comment:
I like this pattern of separating out the unsafe code path into a parameterless method.